package.json

```diff
@@ -42,7 +44,7 @@
       "pre-push": "npm run lint"
     }
   },
-  "bin": "./pokemon-showdown",
+  "bin": "pokemon-showdown",
```

Comment:
?

Reply:
Accidental. Not sure why this keeps happening.